aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,187 @@
+.. _unlock-handlers:
 
-The **Preview** tab shows you the publish path (by default, the same directory as the ".sln" solution file).
+Install the Microsoft ASP.NET Core Module
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Deploy to IIS server
------------------------------------
+Install the Microsoft ASP.NET Core Module version 1.0 or higher on the server:
 
-#. Navigate to the publish folder (*..\\..\\artifacts\\bin\\WebApp9\\Release\\Publish folder* in this sample).
-#. Copy the **approot** and **wwwroot** directories to the target IIS server. Note: MSDeploy is the recommended mechanism for deployment, but you can use Xcopy, Robocopy or another approach. For information on using `Web Deploy` see :doc:`iis-with-msdeploy`.
-#. In IIS Manager, create a new web site and set the physical path to **wwwroot**. You can click on **Browse *.80(http)** to see your deployed app in the browser.
+- `AspNetCoreModule x64 MSI Installer <http://download.microsoft.com/download/7/2/D/72DD68D5-7386-4A41-8B8F-E21779A69E33/aspnetcoremodule_x64_en.msi>`_
+- `AspNetCoreModule x86 MSI Installer <http://download.microsoft.com/download/7/2/D/72DD68D5-7386-4A41-8B8F-E21779A69E33/aspnetcoremodule_x86_en.msi>`_
 
-.. image:: pubIIS/_static/b8.png
+Install .NET Core on the Server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-IIS server configuration
------------------------------------
+If you are planning to host all of your applications as self-contained applications on your server (i.e., you will not deploy portable applications that require installing .NET Core on the server), you may skip this step. For more information on the difference between portable applications and self-contained applications, see `Types of portability in .NET Core <https://github.com/blackdwarf/core-docs/blob/clidocs/docs/core-concepts/app-types.md>`_.
 
-1. Enable the **Web Server (IIS)** server role. In client operating systems (Windows 7 through Windows 10) select **Control Panel > Programs > Programs and Features > Turn Windows features on or off**, and then select **Internet Information Services**.
+Visit the `GitHub dotnet/cli repo (rel/1.0.0 branch) <https://github.com/dotnet/cli/tree/rel/1.0.0>`_ to obtain the correct MSI installer or binaries. To install .NET Core using the MSI, obtain the correct **.NET Core Installer** for your server. If you prefer to install from binaries, obtain the correct **.NET Core Binaries** for your server. These selections will only install .NET Core assemblies and not the full SDK with tools. If you plan to work with the CLI tools on the server, install the SDK version of the MSI or binaries, which includes both tools and the .NET Core assemblies.
 
-.. image:: pubIIS/_static/rs.png
+.. note:: If you install .NET Core from binaries, either include the path to `dotnet.exe` in the system PATH environment variable or make sure that your `web.config` file `\<aspNetCore\>` element `processPath` attribute includes the full path to `dotnet.exe` on the server.
```

Comment:
The reason I mentioned the PATH was that I thought you were still going to make the binaries avaiable. If I'm hearing you correctly, the binaries for the shared framework are going away in favor of the bundle. Therefore, I will remove the bit about the PATH.

I will add here that `iisreset` must be issued or the server should be restarted after the bundle is installed.